pages/windows/setx.md

```diff
@@ -0,0 +1,20 @@
+# setx
+
+> Sets persistent environment variables.
```

Comment:
What about removing `persistent`? 😇 
```suggestion
> Sets environment variables.
```

Reply:
I added this on purpose, because the normal `set` inside `cmd` only sets transient environment variables that are only effective during that console session.

Reply:
OK, so I would say that it:
> Sets environment variables

And that some commands `Set transient environment variables (only effective during that console session)...`
😇